### PR TITLE
Pull Request - Add BasePopup (task#8)

### PR DESCRIPTION
**Pull request by 96tm**
Completed tasks: <a href="https://trello.com/c/e8vg8VH3">Base Popup (task#8)</a>
Screenshot from 2021-09-07 20-39-37
Screenshot from 2021-09-07 20-55-27

### CODE REVIEW

File: src/pages/shared/base-popup/base-popup.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    headingText,
+    buttonOkText,
+    buttonCancelText,
+    className,
+    buttonOkProps,
+    buttonCancelProps,
+    contentProps,
+    children,
+  } = props;
+  const popupStyle = className ? styles[String(className)] : '';
```
Comment (Dasha1801) [inline src/pages/shared/base-popup/base-popup.tsx:29]: test

**Review by Dasha1801 — APPROVED** [review]
test

File: src/shared/constants.ts [hunk 2]
```diff
@@ -5,3 +5,10 @@ export enum TStatusCode {
   BAD_REQUEST = 400,
   ACCEPTED = 202,
 }
+
+const URL_REGEXP =
+  '^(http://www.|https://www.|http://|https://)[a-z0-9]+([-.]{1}[a-z0-9]+)*.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?$';
```
Comment (AlexandrKlychnikov) [inline src/shared/constants.ts:10]: Too match string better make constant

**Review by AlexandrKlychnikov — APPROVED** [review]
Ok

**Pull request merged**